src/nemo_evaluator/solvers/harbor.py

```
@@ -1453,6 +1542,28 @@ def _configure_terminus_tokenizer(*, is_terminus2: bool, model_name: str, tokeni
         _register_harbor_tokenizer(model_name, tokenizer)
 
 
+def _count_zero_token_agent_turns(trajectory: Any) -> int:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `trajectory`

(ANN401)